cartridge/pool.lua

```diff
@@ -138,7 +138,13 @@ local function _pack_values(maps, uri, ...)
     end
 end
 
-local function _gather_netbox_call(maps, uri, fn_name, args, opts)
+local function _gather_netbox_call(fiber_storage, maps, uri, fn_name, args, opts)
+    local self_storage = fiber.self().storage
+    table.clear(self_storage)
```

Comment:
Actually after https://github.com/tarantool/tarantool/issues/3462 you don't need this step

Reply:
Removed.